### ISSUE TITLE
Option to disable custom attributes & classes from control panel

### ISSUE THREAD

**Issue by acalvino4** [verbb/navigation#288]
**What are you trying to do?**
Currently Navigation has the flexibility to add custom attributes to a node via the control panel. This is great; however, usually I don't want content editors to have the ability to edit this for a couple reasons.
* It clutters their interface, making their job (editing content) harder
* It gives potential for them to mess things up (i.e. I have a `id` attribute in my template code, and a content author overrides it for some dumb reason)

I actually don't really want developers do have access to setting node attributes here either, because I don't want them tempted to make style changes (i.e. via classnames) that get stored in the db and can't be easily tracked in git.

**What's your proposed solution?**
The 'workaround' is to deal with it - which is fine - but the option to control this would be nice. I imagine an implementation could go one of two ways (or maybe even a mix of the two), but my preference would be the latter
* Making these fields only show up depending on a user's permissions
* Making these fields only show up depending on settings for a given navigation

Another idea to go with this would be to provide a version of `node.linkAttributes` that is a little more restricted in that it only outputs attributes that are specified at a high level, such as the linked url, whether it get's opened in a new window, whether an asset time should be downloaded - i.e. the things that aren't specified directly at the `attributeName = attributeValue` level, which I would prefer to keep in my versioned templates.

As an example, I actually created my own macro to do this, because
```html
<a {{ node.linkAttributes }} class="h-6 w-6 text-gray-400 hover:text-gray-500">
```
overwrote my class names with an empty `class`. Now my code looks like 
```html
<a {{ linkAttributes(node) }} class="h-6 w-6 text-gray-400 hover:text-gray-500">
```
where my macro is defined as
```twig
{% macro linkAttributes(node) %}
  href="{{ node.url }}"
  {% if node.newWindow %}
    target="_blank" rel="noopener"
  {% endif %}
{% endmacro %}
```

Basically I'm just proposing that this would be built into the plugin as an attribute of `node` (maybe called `coreLinkAttributes`, idc), since I have a hunch that I'm not the only purist that wants to keep the cp to just content and leave code in the code. Or, if there is setting like a proposed to just disable additional attributes from the control panel, that would be even better, since it wouldn't give users the feeling that they are setting something when it just ends up being ignored by the template code. In this case I don't think I would need `coreLinkAttributes` since they are the only ones that are allowed to be set.

P.S. I know I can pass additional attributes to [linkAttributes](https://verbb.io/craft-plugins/navigation/docs/developers/node#linkAttributes), but [this did not seem to render correctly](https://github.com/verbb/navigation/issues/287), and even if it did, it is unclear whether the attributes would be merged appropriately with what is set in the cp.

**Comment by engram-design** [comment]
We're working on Navigation 2.0 for Craft 4 which will do away with the modal HUD in favour of the slide-out you see when double-clicking on an entry. This'll give us a bit more breathing room, and allow us to split these sorts of things to "advanced". But it's also a great idea to be able to disable these things altogether.

**Comment by acalvino4** [comment]
Oh nice! I was thinking about sending another feature request about the editing interface, but sounds like you're on it. Overall, its already a great, flexible plugin, but thanks for putting in the work to bring it to the next level!

**Comment by engram-design** [comment]
This has been implemented in [2.0.0-beta.4](https://github.com/verbb/navigation/releases/tag/2.0.0-beta.4) for Craft 4

**Comment by acalvino4** [comment]
Just curious if your team has had any more thoughts or decisions on allowing us to disable the 'advanced' tab.
